gatsby-config.js

```diff
@@ -48,6 +48,9 @@ module.exports = {
           contentPadding: '2rem',
           maxWidth: '1600px',
         },
+        i18n: {
+          additionalLocales: [{ name: 'Japanese', locale: 'jp' }],
```

Comment:
Now that I think of it, we should probably copy the text from the docs site to use as the name

<img width="138" alt="Screen Shot 2021-01-06 at 10 22 13 AM" src="https://user-images.githubusercontent.com/565661/103805873-0f816c80-5009-11eb-90fc-5b445acde9a6.png">

Reply:
You're very right!! good idea, I'll make that change